slash-commands/moderation/kick.js

```diff
@@ -0,0 +1,92 @@
+const Discord = require('discord.js');
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {promisePool} = require('../../config/db');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {
+  isServerStaff,
+  sendNoTargetStaffReply,
+} = require('../../helpers/validation');
+const {verifyReasonLength} = require('../../helpers/stringHelpers');
+
+const kickIntro = "You've been kicked for the following reason: ```";
+const kickOutro = ' ```';
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('kick')
+    .setDefaultPermission(false)
+    .setDescription('Kick a user')
+    .addUserOption((option) =>
+      option.setName('target').setDescription('The user').setRequired(true)
+    )
+    .addStringOption((option) =>
+      option.setName('reason').setDescription('The reason').setRequired(true)
+    ),
+  async execute(interaction) {
+    const targetUser = await interaction.guild.members.cache.get(
+      await interaction.options.getUser('target')?.id
+    );
+    if (!targetUser) {
+      return interaction.reply('The specified user could not be found.');
+    }
+
+    if (await isServerStaff(interaction, targetUser)) {
+      return await sendNoTargetStaffReply(interaction);
+    }
+
+    const reason = await interaction.options.getString('reason');
+    if (!verifyReasonLength(reason, interaction)) return;
+
+    const kickText = kickIntro + reason + kickOutro;
+
+    try {
+      await dmUser(interaction, targetUser, kickText);
+      await targetUser.kick(kickText);
+      await interaction.channel.send(`${targetUser} was kicked`);
+      await recordKickInDB(interaction, targetUser, reason);
+      await sendToAuditLogsChannel(interaction, {
+        color: '#0099ff',
+        titleMsg: `${targetUser.user.username}#${targetUser.user.discriminator} was kicked by ${interaction.user.tag}:`,
```

Comment:
Instead of `${targetUser.user.username}#${targetUser.user.discriminator}`, we can just use `${targetUser.user.tag}`. There is also a colon at the end of this that shouldn't be there.

Reply:
Good to know. I made the change